eventhandling/configureEvent.go

```diff
@@ -31,13 +30,20 @@ const ErrorRate = "error_rate"
 const ResponseTimeP50 = "response_time_p50"
 const ResponseTimeP90 = "response_time_p90"
 const ResponseTimeP95 = "response_time_p95"
-const keptnPrometheusSLIConfigMapName = "prometheus-sli-config"
-const podNamespaceEnvName = "POD_NAMESPACE"
 const metricsScrapePathEnvName = "METRICS_SCRAPE_PATH"
 const environmentEnvName = "env"
 
 const sliResourceURI = "prometheus/sli.yaml"
 
+var (
```

Comment:
While this is technically correct how you are reading the environment variables, can you please change it to the way it is implemented here: https://github.com/keptn/keptn/blob/master/distributor/cmd/main.go#L47

* The benefit is that you can set a default value. 
* Please use CamelCase for variable names.

Reply:
Done